<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChessForm</class>
 <widget class="QWidget" name="ChessForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lbl1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLCDNumber" name="LCD1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_a">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>54</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout"/>
   </item>
   <item row="0" column="2">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lbl2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLCDNumber" name="LCD2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QFrame" name="frame_3">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>611</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ip</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ip">
         <property name="text">
          <string>192.168.0.106</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="port">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>9000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>用户名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fromname">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>对手</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="toname">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_bind">
         <property name="text">
          <string>绑定</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>471</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="P_VS_P">
         <property name="text">
          <string>人人对战</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="P_VS_C">
         <property name="text">
          <string>人机对战</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="N_VS_N">
         <property name="text">
          <string>网络对战</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>test</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
